aiida/cmdline/params/options/commands/computer.py

```diff
@@ -109,6 +138,17 @@ def should_call_default_mpiprocs_per_machine(ctx):  # pylint: disable=invalid-na
     'Use 0 to specify no default value.',
 )
 
+DEFAULT_MEMORY_PER_MACHINE = OverridableOption(
+    '--default-memory-per-machine',
+    prompt='Default amount of memory per machine (KB).',
```

Comment:
would you mind changing to kB (lower case 'k')?

Reply:
no problem.